README.md

```
@@ -102,4 +103,4 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for requirements and details.
 
 ---
 
-**🌟 Star this repo to track the latest deals | 🔗 Share with your network**
\ No newline at end of file
+**🌟 Star this repo to track the latest deals | 🔗 Share with your network**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)